MapleServer2/PacketHandlers/Game/FieldEnterHandler.cs

```diff
@@ -38,14 +44,40 @@ public class FieldEnterHandler : GamePacketHandler {
             }
 
             // Add catalysts for testing
-            /*
-            int[] catalysts = { 40100001, 40100002, 40100003, 40100021, 40100023, 40100024, 40100026 };
-            foreach (int catalyst in catalysts) {
-                var item = new Item(catalyst) { Amount = 99999, Uid = catalyst };
-                session.Inventory.Add(item);
-                session.Send(ItemInventoryPacket.Add(item));
+
+            int[] catalysts = { 40100001, 40100002, 40100003, 40100021, 40100023, 40100024, 40100026, 15100216, 15100216 };
+
+            // Adds first set of item test case
+
+            Item item = new Item(40100001)
+            {
+                Amount = 10,
+                Uid = BitConverter.ToInt64(Guid.NewGuid().ToByteArray(), 0)
+            };
+            Item item2 = new Item(40100001)
+            {
+                Amount = 8,
+                Uid = BitConverter.ToInt64(Guid.NewGuid().ToByteArray(), 0)
+            };
+            Item item3 = new Item(40100001)
+            {
+                Amount = 5,
+                Uid = BitConverter.ToInt64(Guid.NewGuid().ToByteArray(), 0)
+            };
+
+            session.Inventory.Add(item);
+            session.Inventory.Add(item2);
+            session.Inventory.Add(item3);
+
+            Debug.WriteLine(item.Amount);
+
+            session.Send(ItemInventoryPacket.Add(item));
+            if (item2.Amount > 0)
+            {
+                Debug.WriteLine(item2.Amount);
+                session.Send(ItemInventoryPacket.Add(item2));
+                session.Send(ItemInventoryPacket.Add(item3));
             }
```

Comment:
Doesn't make sense that `item3` is only added if `item2.Amount > 0`. Also an `InventoryController` of sorts should determine which inventory packets should be sent.

Reply:
I just added item 3 to test if it would stack correctly in a subsequent stacking situation. 

IE. 1st item is stacked to full, 2nd item is stacked to full. 3rd item is left over.